### PR TITLE
Search logic

### PR DESCRIPTION
**Pull request by sudo-joseph**
Updates search bar to call search API then show results in cards and on map.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -66,11 +72,59 @@ fetchFacilities = () => {
...
+    })
+    .then((materials_data) => {
+      let materials = [];
+      materials_data.results.result.map(material=> {
+        materials.push({value:material.material_id,
+                        label:material.description})
+      });
+      this.setState({materials:materials})
+      });
+    this.render()
```
Comment (ishinan) [inline client/src/App.js]: `this.render()` comes after then() without then()? Or, was `this.render()` supposed to be a call back in setState()?

**Review by ishinan — DISMISSED** [review]
Another great achievement.

File: client/src/App.js (hunk 1 above)
Comment (sudo-joseph) [reply client/src/App.js]: Yeah, you are correct. Updated, along with a few others that were done like this.

**Review by ishinan — APPROVED** [review]
Super! Great progress again.

**Pull request merged**